tests/unit/test_manifests.py

```diff
@@ -8,7 +8,40 @@
 
 import pytest
 
-from manifests import SetCNIPath
+from manifests import SetCNIPath, UpdateDaemonSetTolerations
+
+
+def test_update_tolerations(harness):
+    harness.disable_hooks()
+    harness.begin_with_initial_hooks()
+    patch = UpdateDaemonSetTolerations(harness.charm.manifests)
+    toleration = mock.MagicMock()
+    toleration.key = "node-role.kubernetes.io/important"
+    toleration.operator = "Exists"
+    toleration.effect = "NoSchedule"
+
+    obj = mock.MagicMock()
+    obj.kind = "DaemonSet"
+    obj.metadata.name = "kube-sriov-cni-ds-amd64"
+    obj.spec.template.spec.tolerations = [toleration]
+    patch(obj)
+    assert len(obj.spec.template.spec.tolerations) == 2
+    assert any(
+        t.key.endswith("control-plane") for t in obj.spec.template.spec.tolerations
+    )
+    assert any(t.key.endswith("important") for t in obj.spec.template.spec.tolerations)
```

Comment:
```suggestion
    assert any(
        t.key.endswith(("control-plane", "important")) for t in obj.spec.template.spec.tolerations
    )
```
`str.endswith` accepts a tuple. What do you think about refactor the two asserts into one?

Reply:
🤯 I WAS TODAY YEARS OLD when i learned this